### ISSUE TITLE
(1.19.2) Unable to Load/Creative Install Modules

### ISSUE THREAD

**Issue by HyperMan19680** [lehjr/MachineMusePowersuits#92]
**Version info**
Java version: Java is version 17.0.3.1, using 64-bit architecture, from Oracle Corporation.
Minecraft Forge version: 1.19.2's 43.3.5

**Describe the bug**
In Creative Mode, I cannot add any modules except a power battery to the Modular arm, and only if I don't choose another item slot first

**To Reproduce**
1.Open Module Bench with Arm currently equipped
2. Problem below

**Expected behavior**
Basically what is doable in 1.16.5's versions that I played before, being able to install modules in Creative Mode.

**Mod list**
Modular Powersuits 1.19.2-2.3.6
Numina 1.19.2-2.3.6

**Screenshots**
https://prnt.sc/_sfV0dbwxY8j

**Additional context**

**Comment by lehjr** [comment]
Going need more than that. Most likely this is an issue with another mod, so I'm going need a list of installed mods.

**Comment by HyperMan19680** [comment]
About 200+ of them, is there a quicker way than just listing off name and version of every mod I have?

**Comment by lehjr** [comment]
No, not unless you're using a modpack.

**Comment by lehjr** [comment]
The thing is, I can't diagnose and fix what I can't reproduce.

**Comment by HyperMan19680** [comment]
I am well aware. I can /probably/ put together a modpack on CurseForge if it helps.

**Comment by HyperMan19680** [comment]
Apologies for the wait, its my first time publshing a modpack on CurseForge's app, so I'm a bit lost.

**Comment by HyperMan19680** [comment]
I decided to put it on my OneDrive  and link it here. https://1drv.ms/u/s!AgUnhOcws_SIia8CBfMTDvJxqyzDeQ?e=mbKgKO

Those /should/ be the mod files I have ATM.

**Comment by HyperMan19680** [comment]
Update, can confirm the next update partially fixes the problem. I can add modules, but not by the Creative Mode method, I can shift-click them in and they'll head to the right spot in the current item.

**Comment by lehjr** [comment]
Yeah, that was to fix a bug I found in the packet code, but there's still something going on with one of the mods. I'm currently working on updating the code for the overlay (HUD, meters, etc) so once I'm done with that I'll have another look at it. I want to add an option to "add all" anyway, since adding one at a time is a pain. If you have any issues adding modules manually, you can add them in a crafting grid with the modular item and get the result with the module installed.

**Comment by HyperMan19680** [comment]
Very well. I can potentially wait for a release version, if that helps.

**Comment by lehjr** [comment]
Try disabling iron jetpacks

**Comment by HyperMan19680** [comment]
Iron Jetpacks was the problem. You got the ace in the hole there. I removed it, and the items came back. :)

**Comment by Stryker944** [comment]
Hi,
first i want to thank you. Its a great mod i really enjoy to play it, but i think i have the same problem like HyperMan.
I can´t install modules. It´s possible to select the modules in the tinkertable and install it.

I uploaded my installed mods to workuplad. I hope you can help me.
[https://workupload.com/file/zPWH3nxRUnX]

**Comment by eyeonus** [comment]
Try disabling your mods one at a time and seeing if the problem is fixed. If it isn't, re-enable that mod and disable the next one in the list.

**Comment by HyperMan19680** [comment]
It would also help if you went and made your own report instead of hijacking someone else's
